pkg/controller/migration/convert/components_test.go

```diff
@@ -0,0 +1,169 @@
+package convert
```

Comment:
I wonder if this file should be called `testutils_test.go` ...

Reply:
Yeah I think a different name would be good.